src/popup/components/ConnectWalletForm.tsx

```diff
@@ -76,6 +97,31 @@ export const ConnectWalletForm = ({ publicKey }: ConnectWalletFormProps) => {
       })}
       className="space-y-4"
     >
+      {errors.root && (
```

Comment:
Looks like this
![image](https://github.com/interledger/web-monetization-extension/assets/8426945/0d2d0081-9bdc-4861-9de6-b03b0fea3421)

Reply:
This is only going to happen if the user installs the extension from Chrome Web Store or Mozilla Add-ons and declines the popup for permissions?

Reply:
It's going to happen even with load-unpacked extension. And we can't ask for the hosts permission at install time. So, this will be visible as whenever we don't have the permissions. We should perhaps add another icon or reuse some existing when we don't have the permissions to show action is _disabled_, and prompt user to give permission.

Reply:
having a scroll bar in the popup I would say is a pretty bad UX, maybe just display this message, and just after that let the user config things? To be discussed with the team/designer

Reply:
Show only the message when missing permissions?

Reply:
The UI to be addressed in a dedicated ticket